### PR TITLE
Mc/scss styling components

### PR DESCRIPTION
**Pull request by Maia-Cochran**
## Pull request type
Describe the purpose of this PR(other):
- adds mock password input, no auth
- adds styling to components
- refactors divs/p-tags/etc. in components

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please describe):

## Problem/feature

What problem are you trying to solve?
- improve UX through styling
- no password input

## Solution

_How did you solve the problem?_
- add password input
- add divs/move flex grid and flex boxes

Any other comments, questions or concerns?
- Circle CI still broken???

## Checklist
- [x] Code compiles correctly
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/components/FavoritedReportCard/FavoriteReportCard.js [hunk 1]
```diff
@@ -2,18 +2,37 @@ import React from 'react';
 import './FavoriteReportCard.scss';
 import PropTypes from 'prop-types'
 
-const FavoriteReportCard = ({ name, studentTeacherRatio, instructionSalaryPercentOfTotal, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
+const FavoriteReportCard = ({ name, studentTeacherRatio, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
```
Comment (edjrodriguez) [inline src/components/FavoritedReportCard/FavoriteReportCard.js]: why did we get rid of this instructional salary percent of total.  We agreed to display this yesterday.

File: src/components/NavBar/NavBar.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const NavBar = ({ signOut, data}) => {
         return(
             <div className='nav-bar'>
                 <div className='logo-and-buttons'>
-                    <button onClick={ () => navigate('/home') }className='logo'>
+                    <button onClick={ () => navigate('/') }className='logo'>
```
Comment (edjrodriguez) [inline src/components/NavBar/NavBar.js]: navigate back to the overview component?

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const NavBar = ({ signOut, data}) => {
             return(
                 <div className='nav-bar'>
                 <div className='logo-and-buttons'>
-                    <button onClick={ () => navigate('/home') } className='logo'>
+                    <button onClick={ () => navigate('/') } className='logo'>
```
Comment (edjrodriguez) [inline src/components/NavBar/NavBar.js]: navigate back to the overview component?

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -37,11 +37,9 @@ const NavBar = ({ signOut, data}) => {
             return(
                 <div className='nav-bar'>
                     <div className='logo-and-buttons'>
-                        <button onClick={ () => navigate('/home') }className='logo'>
+                        <button onClick={ () => navigate('/') }className='logo'>
```
Comment (edjrodriguez) [inline src/components/NavBar/NavBar.js]: navigate back to the overview component?

File: src/components/ReportCard/ReportCard.js [hunk 5]
```diff
@@ -2,17 +2,38 @@ import React from 'react';
 import './ReportCard.scss';
 import PropTypes from 'prop-types'
 
-const ReportCard = ({ districtName, studentTeacherRatio, instructionSalaryPercentOfTotal, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
+const ReportCard = ({ districtName, studentTeacherRatio, perTeacherSalaryExpenses, enrollment, numberOfSchoolsInDistrict, studentGuidanceCounselorRatio, perStudentExpenditure }) => {
```
Comment (edjrodriguez) [inline src/components/ReportCard/ReportCard.js]: why did we get rid of this instructional salary percent of total. We agreed to display this yesterday.

File: src/components/UserLoginForm/UserLoginForm.js [hunk 6]
```diff
@@ -31,8 +33,16 @@ const UserLoginForm = ({ submitLogin }) => {
...
+                className='password-input'
+                type="text"
+                name="userPassword"
+                placeholder="Enter your password"
+                value={userPassword}
+                onChange={() => setUserPassword("welcome2022")}
+            />
 
-            <button className='login-button' type='submit' disabled={!userCredentials}>Login</button>
+            <button className='login-button' type='submit' disabled={!userCredentials && !userPassword}>Login</button>
```
Comment (edjrodriguez) [inline src/components/UserLoginForm/UserLoginForm.js:45]: this is requiring a password to sign in...  you said in slack that this isnt working yet so I dont know if we should do this especially for testing purposes.

**Review by edjrodriguez — CHANGES_REQUESTED** [review]
Hey I got a few questions about some of these changes made.  I dont agree with the password input while patrick is still trying to make testing happen.  also why did we get rid of instructional salary percetn of total?

File: src/components/FavoritedReportCard/FavoriteReportCard.js (hunk 1 above)
Comment (Maia-Cochran) [reply src/components/FavoritedReportCard/FavoriteReportCard.js]: It was not rendering any information, so I assumed this was legacy code. My apologies!

File: src/components/UserLoginForm/UserLoginForm.js (hunk 6 above)
Comment (Maia-Cochran) [reply src/components/UserLoginForm/UserLoginForm.js:45]: This is working, but it doesn't include any auth which was agreed upon as a group.

**Review by edjrodriguez — APPROVED** [review]

**Pull request merged**